travis.sh

```diff
@@ -7,6 +7,11 @@ set -eu
 bundle check || bundle install
 
 case "$TRAVIS_EVENT_TYPE" in
+  pull_request)
+    echo "Running tests and not posting anywhere"
+    bundle exec rspec
+    REALLY_POST_TO_SLACK="0" bundle exec rake notify:expired
+    ;;
```

Comment:
[This commit](https://github.com/alphagov/tech-docs-monitor/commit/e6e38d8139c852b35a9029ffb3fceb410193e550) makes it such that Daniel posts to:
- configured documentation owners when the cron job runs
- #bot-testing on Slack when on PR and merge

I think it's useful to have Daniel the Manual Spaniel post to #bot-testing when running tests. Why silence Daniel on pull requests?

Reply:
This was to fix that when the [job fails](https://travis-ci.com/alphagov/tech-docs-monitor/jobs/273482748) because `SLACK_WEBHOOK_URL` isn't found. My thinking was that you'd want Travis to build a PR but probably not send to Slack, otherwise anyone publicly could (maliciously or otherwise) raise a PR that could spam our channels. Once someone has approved a PR and merged, it would make sense to send to `#bot-testing`.

Reply:
Yeah that's a good point - we should disable Travis runs for PRs that have come from a fork to avoid the danger of malicious spam. I think it's still useful for GDS users to make sure their Slack notifications are working.

I'm not sure why the job was failing because of `SLACK_WEBHOOK_URL`, that should always be present as it's encrypted in the `.travis.yml`. :thinking:

Reply:
Oh, I didn't see this bit in the Travis docs:

> Travis CI makes encrypted variables and data available only to pull requests coming from the same repository. These are considered trustworthy, as only members with write access to the repository can send them.

Reply:
Yeah definitely @vixus0. What's the difference between the `pr` and `pull_request` build types? I'm happy to can the changes to `travis.sh` in this PR and raise something separate?